hathor/builder/cli_builder.py

```diff
@@ -100,6 +100,7 @@ def create_manager(self, reactor: Reactor) -> HathorManager:
             python=python,
             platform=platform.platform(),
             settings=settings_source,
+            reactor_type=type(reactor),
```

Comment:
Maybe `repr()` works best since it can show the initialization parameters?

Reply:
I'm even OK even with just `reactor_type=type(reactor).__name__`.

Reply:
Got it. I'm ok either way here. You choose. ;)

Reply:
I changed it to @jansegre's suggestion in https://github.com/HathorNetwork/hathor-core/pull/889/commits/15ba731c56ee99802a0021d0e6791c66441ed121

Reply:
Thanks!